examples/02-Advanced-workflow.ipynb

```diff
@@ -0,0 +1,1060 @@
+{
```

Comment:
<p>We should add a next steps to other notebooks with more Advanded Workflows.</p><p> </p><p>Next steps:</p><p>We are using NVTabular in our Merlin repositories to preprocess and engineer features before training. We can recommend multiple examples, which shows complex pipelines with NVTabular:</p><p>- https://github.com/NVIDIA-Merlin/models/blob/main/examples/usecases/ecommerce-session-based-next-item-prediction-for-fashion.ipynb</p><p>- https://github.com/NVIDIA-Merlin/Transformers4Rec/blob/main/examples/end-to-end-session-based/01-ETL-with-NVTabular.ipynb</p><p>- https://github.com/NVIDIA-Merlin/publications/blob/main/tutorials/RecSys22tutorial/02-Implementing-RecSys-architectures.ipynb (Add here a note, that this is not maintained)</p><p> </p><p>Can you add a short description to each of the example - what is processed, what is relevant for the NVTabular users?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/NVTabular/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
information added